metadata-ingestion/src/datahub/ingestion/source/metabase.py

```diff
@@ -48,8 +48,8 @@ class MetabaseConfig(DatasetLineageProviderConfigBase):
     # See the Metabase /api/session endpoint for details
     # https://www.metabase.com/docs/latest/api-documentation.html#post-apisession
     connect_uri: str = Field(default="localhost:3000", description="Metabase host URL.")
-    username: str = Field(default=None, description="Metabase username.")
-    password: str = Field(default=None, description="Metabase password.")
+    username: NoneStr = Field(default=None, description="Metabase username.")
```

Comment:
Can we use either `Optional[str]` or its pydantic type alias `pydantic.NoneStr` consistently across the configs?